README.md

```diff
@@ -33,15 +33,18 @@ Demonstrate your understanding of this week's concepts by answering the followin
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your Team Lead
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
-
+    Semantic HTML are tags that contain some sort of meaning. 
 2. Name two big differences between ```display: block;``` and ```display: inline;```.
+   - display block presents everything in a block and holds no html elements next to it.
+   - display inline has all the elements display in the current block on the same line.
 
 3. What are the 4 areas of the box model?
+    - conten, padding, border, margin
 
 4. While using flexbox, what axis does the following property work on: ```align-items: center```?
-
+    - this will align all the items on the center of the cross axis going horizontal.
 5. Explain why git is valuable to a team of developers.
-
+    - git is valuable because it is faster and more efficient than using GUI's.
```

Comment:
Git is a tool for "version control" it allows developers to keep track of their code and is a very powerful tool for team development. It allows multiple users to collaborate on one project. You are probably confusing the "Terminal" or "CLI" for git. I suggest to review git since it is an essential part of the lambda school curriculum.